jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -192,6 +194,24 @@ function with_retries() {
   return "$rc"
 }
 
+function check_retry_params() {
```

Comment:
I don't see the need to implement this as command execution wrapper. The parameter validation should be included in function `with_retries` generically (without referring to the `PIPELINE_CLONE_*` env var names in error messages).

Reply:
okay i'll do as suggested